[2025-05-29T00:00:03.092+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: punctuality_etl_dag.transform_data scheduled__2025-05-28T00:00:00+00:00 [queued]>
[2025-05-29T00:00:03.099+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: punctuality_etl_dag.transform_data scheduled__2025-05-28T00:00:00+00:00 [queued]>
[2025-05-29T00:00:03.101+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-05-29T00:00:03.178+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transform_data> on 2025-05-28 00:00:00+00:00
[2025-05-29T00:00:03.180+0000] {standard_task_runner.py:57} INFO - Started process 20817 to run task
[2025-05-29T00:00:03.182+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'punctuality_etl_dag', 'transform_data', 'scheduled__2025-05-28T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/punctuality_etl_dag.py', '--cfg-path', '/tmp/tmpecjp4ztg']
[2025-05-29T00:00:03.184+0000] {standard_task_runner.py:85} INFO - Job 57: Subtask transform_data
[2025-05-29T00:00:03.214+0000] {task_command.py:415} INFO - Running <TaskInstance: punctuality_etl_dag.transform_data scheduled__2025-05-28T00:00:00+00:00 [running]> on host a5551f46ca35
[2025-05-29T00:00:03.266+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='punctuality_etl_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-28T00:00:00+00:00'
[2025-05-29T00:00:03.277+0000] {logging_mixin.py:151} INFO - Cleaned punctuality data saved to data/processed/punctuality_avg_cleaned.csv
[2025-05-29T00:00:03.277+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-29T00:00:03.284+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=punctuality_etl_dag, task_id=transform_data, execution_date=20250528T000000, start_date=20250529T000003, end_date=20250529T000003
[2025-05-29T00:00:03.322+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-29T00:00:03.339+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
